test/cql-pytest/test_bad_grammar.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Tests for the parser, focusing on rejected grammar
+#############################################################################
+
+import pytest
+from util import new_test_table
+from cassandra.protocol import InvalidRequest, SyntaxException
+
+
+# A collection of tests for grammar that should be rejected, but wasn't at
+# some point in the past.
+
+# table1 has some pre-set data which the tests below SELECT on (the tests
+# shouldn't write to it).
+@pytest.fixture(scope="module")
+def table1(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "p int, c1 int, c2 int, v int, PRIMARY KEY (p, c1, c2)") as table:
+        stmt = cql.prepare(f'INSERT INTO {table} (p, c1, c2, v) VALUES (?, ?, ?, ?)')
+        cql.execute(stmt, [0, 0, 0, 1])
+        cql.execute(stmt, [0, 0, 1, 2])
+        cql.execute(stmt, [0, 1, 0, 3])
+        cql.execute(stmt, [0, 1, 1, 4])
+        cql.execute(stmt, [1, 0, 0, 5])
+        cql.execute(stmt, [1, 0, 1, 6])
+        cql.execute(stmt, [1, 1, 0, 7])
+        cql.execute(stmt, [1, 1, 1, 8])
+        # FIXME: I would like to make the table read-only to prevent tests
+        # from messing with it accidentally.
+        yield table
+
+### GROUP BY without aggregation:
```

Comment:
I don't know what this comment refers to. Copy-pasta?

Reply:
removed